griptape/tasks/base_task.py

```diff
@@ -60,6 +60,26 @@ def meta_memories(self) -> list[BaseMetaEntry]:
     def __str__(self) -> str:
         return str(self.output.value)
 
+    def add_parents(self, *parents: str | BaseTask) -> None:
```

Comment:
I think this should be `parents: list[str] | list[BaseTask]`. That way it won't break if we add  additional positional arguments in the future.

Reply:
I agree I like an explicit list better (though I'd prefer `list[str | BaseTask]`).

I originally chose the variable args syntax for consistency with [Structure.add_tasks()](https://github.com/griptape-ai/griptape/blob/1042211e22e99425ea8867fb9c506bf913f5387a/griptape/structures/structure.py#L190).